### PR TITLE
Feature/alert modal

### PR DESCRIPTION
**Pull request by HIMDanny**
create alert modal for global usage

### CODE REVIEW

**Review by RomanRymarchuk1 — APPROVED** [review]

File: client/src/components/AlertModal/AlertModal.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  DialogContentText,
+  DialogTitle,
+  styled,
+} from '@mui/material';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = styled(MuiButton)({
+  padding: '12px 24px',
+});
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/components/AlertModal/AlertModal.js:15]: no need for that,   just pass the padding in sx prop in the Button itself

File: client/src/components/AlertModal/AlertModal.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+AlertModal.propTypes = {
+  open: PropTypes.bool.isRequired,
+  onClose: PropTypes.func.isRequired,
+  onSubmit: PropTypes.func.isRequired,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+};
+
+AlertModal.defaultProps = {
+  title: '',
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/components/AlertModal/AlertModal.js]:
```suggestion
  title: 'Alert !',
```

File: client/src/components/AlertModal/AlertModal.js (hunk 1 above)
Comment (HIMDanny) [reply client/src/components/AlertModal/AlertModal.js:15]: I think it is better to use one styled component than writing to more than one component same stylings

Comment (Ahmed-Fahmy-Dev) [reply client/src/components/AlertModal/AlertModal.js:15]: i dont agree in this use case but not a deal breaker.

**Review by Ahmed-Fahmy-Dev — APPROVED** [review]

**Pull request merged**